apps/modernization-ui/src/components/button/nav/NavLinkButton.tsx

```diff
@@ -1,22 +1,24 @@
 import { RefAttributes } from 'react';
-import { NavLink, NavLinkProps } from 'react-router-dom';
+import { NavLink, NavLinkProps, To } from 'react-router-dom';
 import classnames from 'classnames';
 
 type Props = {
     to: string;
-    label?: string;
+    label?: string & To;
```

Comment:
question:  Should the type of `to` be `To`?

Reply:
I made this change earlier on in the development of this feature, just tested and I can remove the 'To' type.